client/projector/Program.js

```diff
@@ -9,6 +9,7 @@ import styles from './Program.css';
 
 function matrixToCssTransform(matrix) {
   /* eslint-disable prettier/prettier */
+  // prettier-ignore
```

Comment:
Is the `eslint-disable` above now redundant?

Reply:
just checked, yup!

(didn't mean to include this change here since linting is already failing in a couple other spots.  can fix in another pr)